### ISSUE TITLE
getUserMedia threw exception :TypeError: navigator.getUserMedia is not a function

### ISSUE THREAD

**Issue by pauljuneau** [cwilso/PitchDetect#34]
Hello. Your app is really cool. It was working for me on my Windows 10 PC the other day via Chrome, and now it no longer works. I received the error provided in the title when I when to your app site, http://webaudiodemos.appspot.com/pitchdetect/, and clicked on the 'use live input' button.   I tried using Microsoft Edge and the issue still occurred there as well.

**Comment by karx** [comment]
Hi @pauljuneau. I looked into why was this happening, and the thing is:

> Chrome is requiring secure origins (HTTPS) for getUserMedia.
> 
> Starting with Chrome 47, getUserMedia() requests are only allowed from secure origins: HTTPS or localhost.
> 
> https://developers.google.com/web/updates/2015/10/chrome-47-webrtc?hl=en
I found many discussions regarding the same including here: https://github.com/WebDevSimplified/Face-Detection-JavaScript/issues/9

If you simply use `https` instead of `http` for the site, it should work like a charm. 
In the repo I see that the README links to `https` version of the site, however the url provided for project in GitHub sidebar is `http` version. Maybe the maintainers can update it to `https`
